src/AstroBase.jl

```diff
@@ -1,5 +1,23 @@
 module AstroBase
 
-# package code goes here
+using Rotations
+export polar_motion
 
-end # module
+"""
+    polar_motion(rx, ry, sp)
+
+Form the matrix of polar motion for coordinates of the pole (radians).
+```
```

Comment:
See my comment on the other PR. See here how doctests work: https://juliadocs.github.io/Documenter.jl/stable/man/doctests/